src/eredis_cluster_pool.erl

```diff
@@ -33,7 +33,8 @@ create(Host, Port) ->
             PoolArgs = [{name, {local, PoolName}},
                         {worker_module, eredis_cluster_pool_worker},
                         {size, Size},
-                        {max_overflow, MaxOverflow}],
+                        {max_overflow, MaxOverflow},
+                        {strategy, fifo}],
```

Comment:
Should put it into application environment?

Reply:
Good idea.